product-payment.html

```diff
@@ -78,93 +78,107 @@
   </header>
   <main>
     <section class="product-page">
-      <div class="product-page__top">
-        <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-          <path fill-rule="evenodd" clip-rule="evenodd" d="M2 5.99998L8 1.33331L14 5.99998V13.3333C14 14.0697 13.403 14.6666 12.6667 14.6666H3.33333C2.59695 14.6666 2 14.0697 2 13.3333V5.99998Z" stroke="#A6A6A6" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
-          <path d="M6 14.6667V8H10V14.6667" stroke="#A6A6A6" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
-        </svg>
-        <div class="product-page__arrow-right"></div>
-        <p class="product-page__text"> Pitnuras</p>
-        <div class="product-page__arrow-right"></div>
-        <p class="product-page__text"> ... </p>
-        <div class="product-page__arrow-right"></div>
-        <p class="product-page__text"> Pinturas de interior </p>
-        <div class="product-page__arrow-right"></div>
-        <p class="product-page__text"> 625CK1775 </p>
-      </div>
+      <nav class="product-page__top" aria-label="lista ordenada">
+        <ol class="product-page__top-Breadcrumb">
+          <li>
+            <a href="index.html" aria-label="ir nuevamente al home">
+              <svg class="product-page__top-svg" width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
+                <path fill-rule="evenodd" clip-rule="evenodd" d="M2 5.99998L8 1.33331L14 5.99998V13.3333C14 14.0697 13.403 14.6666 12.6667 14.6666H3.33333C2.59695 14.6666 2 14.0697 2 13.3333V5.99998Z" stroke="#A6A6A6" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                <path d="M6 14.6667V8H10V14.6667" stroke="#A6A6A6" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+              </svg>
+            </a>
+          </li>
+          <div class="product-page__arrow-right"></div>
+          <li>
```

Comment:
Los div no deberian ir entre un li y otro, sino dentro del li. Referencia: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol#technical_summary

`Permitted content | Zero or more <li>, <script> and <template> elements.`

```suggestion
          <li>
            <div class="product-page__arrow-right"></div>

```